pkg/destroy/aws/shared.go

```diff
@@ -117,6 +117,12 @@ func (o *ClusterUninstaller) removeSharedTag(ctx context.Context, session *sessi
 				}
 				result, err := tagClient.UntagResourcesWithContext(ctx, request)
 				if err != nil {
+					if aerr, ok := err.(awserr.Error); ok {
+						switch aerr.Code() {
+						case resourcegroupstaggingapi.ErrorCodeInvalidParameterException:
+							nextTagClients = nextTagClients[:len(nextTagClients)-1]
```

Comment:
I'm not following this line. Why do this? Can you add a comment in the code?

Reply:
turns out I was right the first time. tagClient is added to the last of the list and has to be removed if IPE is returned.